test/e2e/test_supervisor_orchestration.py

```
@@ -669,3 +669,82 @@ def test_supervisor_handoff(self, require_antigravity):
+            agent_profile="developer",
+            session_name=session_name,
+        )
+        try:
+            # Wait for terminal to be ready
+            assert _wait_for_ready(
+                terminal_id, timeout=30
+            ), "Devin CLI did not become ready within 30s"
+
+            # Send a simple task
+            task_message = "echo hello world"
+            resp = requests.post(
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)